app/controllers/idv/personal_key_controller.rb

```diff
@@ -37,6 +37,8 @@ def next_step
         idv_come_back_later_url
       elsif in_person_enrollment?
         idv_in_person_ready_to_verify_url
+      elsif device_profiling_failed?
```

Comment:
Note that there are currently two implementations of the personal key step. I'd expect we'd update both to keep the behaviors in sync.

The other:

https://github.com/18F/identity-idp/blob/836b5ac2c52da8d7829c09a63c427f851b41832d/app/controllers/api/verify/password_confirm_controller.rb#L49-L59